lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -14,16 +14,31 @@ defmodule ExUnit.Filters do
   on the command line) includes a line number filter, and if so returns the
   appropriate ExUnit configuration options.
   """
-  @spec parse_path(String.t()) :: {String.t(), any}
+  @spec parse_path(String.t()) :: {String.t(), Keyword.t()}
   def parse_path(file) do
-    {paths, [line]} = file |> String.split(":") |> Enum.split(-1)
```

Comment:
What do you think about abstracting the concept of "extract line numbers"? so the code here would be something like:

```elixir
case extract_line_numbers(file) do
  {path, []} ->  {path, []}
  {path, line_numbers} -> {path, exclude: [:test], include: Enum.map(lines, &{:line, &1})}
end
```